contract/test/mintStable.js

```diff
@@ -45,7 +52,7 @@ export const makeStableFaucet = ({
   /**
    * Mint a payment in the fee token.
    *
-   * @param {bigint} value the associated brand is implicitly the fee brand
+   * @param {bigint | any[] | CopySet<any> | CopyBag<any>} value the associated brand is implicitly the fee brand
```

Comment:
this shouldn't change. you can't meeting a Payment for a natural number with these input types. You'll have to narrow the type at the callsite instead